docs/source/api/express-middleware.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: "API Reference: expressMiddleware"
+api_reference: true
+---
+
+This API reference documents Apollo Server 4's [Express](https://expressjs.com/) integration, the `expressMiddleware` function.
+
+## `expressMiddleware`
+
+The `expressMiddleware` function enables you to attach Apollo Server to an Express server. 
```

Comment:
I think we should be pretty explicit that `expressMiddleware` expects that in the case of POST requests, the body has already been parsed as JSON onto `req.body`, which generally means using `body-parser`. And that there is no CORS configuration by default but that we recommend you start with using the cors middleware with its default configuration (and if you don't you won't be able to run operations from any web sites other than embedded Sandbox or another UI on the domain itself).